data/RashidatJay.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Rashidat Jimoh",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student|| Technical Writer|| Developer",
+  "avatar": "https://github.com/RashidatJay.png",
```

Comment:
you can add tags here like this
![image](https://user-images.githubusercontent.com/88339569/215160942-bcc9bca6-356a-4375-8a0a-0d58a0a62752.png)

Reply:
I have done this